app/models/miq_server/update_management.rb

```diff
@@ -136,6 +136,7 @@ def enable_repos
         LinuxAdmin::SubscriptionManager.enable_repo(repo, assemble_registration_options)
       rescue AwesomeSpawn::CommandResultError
         update_attributes(:upgrade_message => "failed to enable #{repo}")
+        Notification.create(:type => "enable_update_repo_failed", :options => {:repo_name => repo})
```

Comment:
Looks good.  I don't know if we can bubble up more information out of the exception but that might be too much information.  Either way, that's more than what you need here.